recipes/leptonica/all/conanfile.py

```diff
@@ -116,3 +123,5 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "Leptonica"
         self.cpp_info.names["cmake_find_package_multi"] = "Leptonica"
         self.cpp_info.names['pkg_config'] = 'lept'
+        self.cpp_info.includedirs.append(os.path.join("include", "leptonica"))
```

Comment:
```suggestion
```

If the author does not export that folder, please, do not expose those header in general include.

Reply:
It is required by the upstream recipe to export both include dirs as seen in Leptonica's [cmake](https://github.com/DanBloomberg/leptonica/blob/master/lept.pc.cmake)/[pc](https://github.com/DanBloomberg/leptonica/blob/master/lept.pc.in) files.